src/plone/restapi/services/querystringsearch/get.py

```diff
@@ -14,11 +17,14 @@
     SUPPORT_NOT_UUID_QUERIES = False
 
 
-class QuerystringSearchPost(Service):
```

Comment:
This is a breaking change I guess. Not saying we can not do a breaking. Just mentioning it here...

Reply:
I should scroll down before writing a comment. Please ignore me and sorry for the noise. :)